platform/linux-generic/include/odp/api/plat/packet_inlines.h

```diff
@@ -369,6 +371,22 @@ _ODP_INLINE odp_packet_chksum_status_t odp_packet_l4_chksum_status(odp_packet_t
 	return ODP_PACKET_CHKSUM_OK;
 }
 
+_ODP_INLINE void odp_packet_l3_chksum_insert(odp_packet_t pkt, int insert)
+{
+	_odp_packet_flags_t *flags = _odp_pkt_get_ptr(pkt, _odp_packet_flags_t, flags);
```

Comment:
Perhaps this declare-and-define-in-the-same-line could be used in the earlier commits as well.

Reply:
The difference is that here we take a pointer to the packet flags. In the previous commits I followed the style used in all previous flag accessor functions.